Core/Core/Planner/APICalendarEvent.swift

```diff
@@ -106,22 +106,22 @@ public struct GetCalendarEventsRequest: APIRequestable {
     public let contexts: [Context]?
     public let startDate: Date?
     public let endDate: Date?
+    private let calendar: Calendar
+    private let timeZone: TimeZone
```

Comment:
I would move these private vars after the public vars list.

Reply:
I removed these as they are not needed to be accessible outside of init.